components/consumer/profile/Thrifts.js

```diff
@@ -0,0 +1,183 @@
+import { useState, useEffect } from "react";
+import HeaderComponent from "@/components/utility/Header";
+import SubscriptionModal from "@/components/scoped/SubscriptionModal";
+import ButtonComponent from "@/components/utility/Button";
+// import { useUser } from "@/context/UserContext";
```

Comment:
remove this

Reply:
removed the unnecessary imports and commented code from here